frontend/src/js/components/PageViewer/PageViewer.tsx

```diff
@@ -68,65 +65,77 @@ export const PageViewer: FC<PageViewerProps> = () => {
   }, [handleUserKeyPress]);
 
   const performFind = useCallback(
-    (query: string) =>
-      authFetch(`/api/pages2/${uri}/find?fq="${query}"`)
+    (query: string) => {
+      const params = new URLSearchParams();
+      // The backend will respect quotes and do an exact search,
+      // but if quotes are unbalanced elasticsearch will error
+      params.set("fq", removeLastUnmatchedQuote(query));
+
+      setIsFindPending(true);
+      return authFetch(`/api/pages2/${uri}/find?${params.toString()}`)
         .then((res) => res.json())
-        .then((searchHits) => {
-          setLastPageHit(middlePage);
-          setFindHits(searchHits);
+        .then((highlights) => {
+          setIsFindPending(false);
```

Comment:
would be nice not to do a round-trip to server when emptying search box, but didn't seem super important for this PR. have added to https://trello.com/c/ycfzJavQ/1986-page-viewer-navigate-between-individual-highlights